.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: AppConfiguration-DotnetProvider CI
+
+on:
+  push:
+    branches:
+      - main
+      - preview
+      - release/*
+  pull_request:
+    branches:
+      - main
+      - preview
+      - release/*
+
+permissions:
+  security-events: write
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install .NET
+        run: |
+          pwsh build/install-dotnet.ps1 -RestoreOnly
+
+      - name: Restore
+        run: |
+          pwsh build.ps1 -RestoreOnly
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v3
+        with:
+          languages: 'csharp'
+
+      - name: Dotnet Build
+        run: |
+          pwsh build.ps1
+
+      - name: Dotnet Pack
+        run: |
+          pwsh pack.ps1
+
+      - name: Dotnet Test
+        run: |
+          pwsh test.ps1 || exit 0
```

Comment:
Does ` || exit 0` mean the task will be successful even if test fails? Is that on purpose, as I would expect test failure to fail the CI pipeline.

Reply:
Yea good point, missed that